src/commands/token/voice.js

```diff
@@ -23,11 +23,22 @@ class VoiceTokenGenerator extends TwilioClientCommand {
       process.exit(1);
     }
 
+    let pushCredentialSid = this.flags['push-credential-sid'];
+    if (pushCredentialSid && !validatePushCredentialSid(pushCredentialSid)) {
+      this.logger.error(
+        'Invalid Push Credential SID, must look like CRxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'
+      );
+      process.exit(1);
+    }
+
     let incomingAllow = (this.flags['allow-incoming'] == 'true');
     let voiceGrant = new Twilio.jwt.AccessToken.VoiceGrant({
       outgoingApplicationSid: this.flags['voice-app-sid'],
       incomingAllow
     });
+    if (pushCredentialSid) {
+      voiceGrant.pushCredentialSid;
```

Comment:
I think this should be:
```suggestion
      voiceGrant.pushCredentialSid = pushCredentialSid;
``` 

If I'm not mistaken?